src/main/java/org/folio/circulation/domain/policy/FixedDueDateSchedules.java

```diff
@@ -48,7 +48,7 @@ public static FixedDueDateSchedules from(JsonObject representation) {
   private Predicate<? super JsonObject> isWithin(DateTime date) {
     return schedule -> {
       DateTime from = DateTime.parse(schedule.getString("from"));
-      DateTime to = DateTime.parse(schedule.getString("to"));
+      DateTime to = DateTime.parse(schedule.getString("to")).withTime(23, 59, 59, 0);
```

Comment:
I think it worth moving this logic to `DateTimeUtil` utility.

Reply:
There is already `org.folio.circulation.support.utils.DateTimeUtil#atEndOfTheDay` method but for java date time class. We can add similar but for joda date time.